<?xml version="1.0"?>
<launch>
        <node pkg="chibi21_b" name="local_path_planner" type="local_path_planner" output="screen">
            <param name="RESOLUTION_VELOCITY_NUM" value="5"/>
            <param name="RESOLUTION_OMEGA_NUM" value="5"/>
            <param name="DT" value="0.05"/>
            <param name="ROBOT_FRAME" value="map"/>
            <param name="LINEAR_SPEED_MAX" value="0.7"/>
            <param name="LINEAR_SPEED_MIN" value="0.1"/>
            <param name="ANGULAR_SPEED_MAX" value="3.0"/>
            <param name="LINEAR_ACCL" value="5.0"/>
            <param name="ANGULAR_ACCL" value="5.0"/>
            <param name="COST_HEADING_GAIN" value="12.5"/>
            <param name="COST_VELOCITY_GAIN" value="4.0"/>
            <param name="COST_OBSTACLE_GAIN" value="1.4"/>
            <param name="COST_HEADING_OBS_GAIN" value="200.0"/>
            <param name="PREDICT_TIME" value="3.0"/>
            <param name="ROBOT_RADIUS" value="1.0"/>

            <param name="USE_DUMMY_TOPIC" value="false"/>
            <!-- <remap from="mcl_pose" to="dummy_estimated_pose"/> -->
            <!-- <remap from="roomba/odometry" to="dummy_twist"/> -->
            <!-- <remap from="local_goal" to="dummy_local_goal"/> -->
    </node>

        <!-- <node pkg="chibi21_b" name="dummy_topic" type="dummy_topic" output="screen"> -->
            <!-- <param name="LOCAL_GOAL_X" value="10.0"/> -->
            <!-- <param name="LOCAL_GOAL_Y" value="&#45;2.0"/> -->
            <!-- <param name="LOCAL_GOAL_YAW" value="1.0"/> -->

        <!-- </node> -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find chibi21_b)/config/dummy.rviz" />
</launch>
